### ISSUE TITLE
How does `logContains` work for you?

### ISSUE THREAD

**Issue by fzipi** [microsoft/waflab#32]
Hi! I'm wondering how are you reading logs for testing that the waf is matching.

- from reading the `logContains` method in https://github.com/microsoft/waflab/blob/5a38a39175b04b2b9bf78816bc98ec60dd8f9718/cmd/command/test.go#L304, the match looks only a regexp match in the file
- this will not work unless each test fires up one and just one rule per test. That is not the common case.

👉  what ftw and go-ftw have been doing is reading the log backwards until the time of the request. I don't see that here.

Can you please explain what is your use case, and how does it work for you?

**Comment by Olament** [comment]
Yes, the ```log_contains``` will only works if we run one testcase at the time. It was added for the back-compatible purppose and we later shift our focus on development the black-box testing.

Thank you for pointing that out, we will update the implementation to align with what ftw and go-ftw are doing in the future.

**Comment by fzipi** [comment]
Was this done? Can we close it?
